v4/source/users.yaml

```diff
@@ -1167,6 +1167,102 @@
             userID := "BbaYBYDV5IDOZFiJGBSzkw1k5u"
 
             mfaSecret, resp = Client.GenerateMfaSecret(userID)
+
+  /users/{user_id}/demote:
+    post:
+      tags:
+        - users
+      summary: Demote a user to a guest
+      description: |
+        Convert a regular user into a guest. This is will convert the user
```

Comment:
Convert a regular user into a guest. This is will convert the user into a guest for the whole system while retaining, their existing team and channel memberships.